src/main/kotlin/no/nav/syfo/fastlege/ws/fastlege/FastlegeInformasjonClient.kt

```diff
@@ -37,10 +37,22 @@ class FastlegeInformasjonClient(
                     helsepersonellregisterId = wsgPOnContractAssociation.hprNumber,
                     fastlegekontor = getFastlegekontor(patientGPDetails.gpContract.gpOffice),
                     pasientforhold = getPasientForhold(patientGPDetails.period),
+                    gyldighet = getGyldighet(wsgPOnContractAssociation.valid),
+                    relasjon = Relasjon(
+                        kodeVerdi = wsgPOnContractAssociation.relationship.codeValue,
+                        kodeTekst = wsgPOnContractAssociation.relationship.codeText,
+                    ),
                 )
             }
     }
 
+    private fun getGyldighet(period: WSPeriod): Gyldighet {
```

Comment:
Ligner veldig på getPasientForhold under. Trengs begge eller kunne vi f.eks. hatt et periode-konsept som begge bruker?

Reply:
Enig i det, men for å bruke det i begge objekter så må man gjøre "breaking changes" i DTO-objektet. 

kan evt. starte med et slikt objekt kun for gyldighet?

Reply:
Blir det breaking change å ta i bruk Periode for pasientforhold? felter har jo fortsatt same navn og verdier.

Reply:
Du sier noe der. Jeg skal endret det.